flipper/client.py

```diff
@@ -63,31 +88,45 @@ def list(
 
     @flag_must_exist
     def enable(self, feature_name: str):
+        self._event_emitter.emit(EventType.PRE_ENABLE, feature_name)
```

Comment:
You might want to consider using context managers to call events before/after so that you can cleanup client code. Adds extra layer of complexity, so up to you 🤷‍♂

Reply:
What do you mean exactly?